README.md

```diff
@@ -228,6 +228,9 @@ deploy workloads.
 ### Cluster-wide proxy 
 WMCO does not support adding Windows workloads using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html) config for the OpenShift Container Platform. WMCO will not be able to automatically route proxy connections for Windows workloads.
 
+### Accessing secure registries
+WMCO does not support pulling images from secure private registries on the Windows nodes. It is recommended to use images from public registries.
```

Comment:
Alternatively the images can pre-pulled onto the VM image that is used

Reply:
Or pulled before adding a BYOH instance to the cluster.

Reply:
I would leave out the automation bit. That is up to the user.

Reply:
@mansikulkarni96 This is related to containerd, for 6.0.0+ only, correct?

Reply:
@mburke5678 I believe this was always the case, even with Docker. It's only with a feature request that came in with containerd that we decided to document it.